programs/castle-vault/src/asset_container/reserves.rs

```diff
@@ -100,13 +104,13 @@ mod test {
         let mut mock_rc1 = MockReturnCalculator::new();
         mock_rc1
             .expect_calculate_return()
-            .return_const(Ok(Rate::from_percent(10)));
+            .return_once(move |_, _| Ok(Rate::from_percent(10)));
```

Comment:
Can you add a comment on the `move` here?

Reply:
@yf-castel for some reason i got a `Result<Rate> does not implement clone` error after the upgrade. tbh i have no idea how move fixes this but it does. added a comment to this effect.